Project/DAQ_System/Program/Src/Sensor/GyroScope/LSM6DSOXGyroscope.hpp

```diff
@@ -0,0 +1,129 @@
+/*
+* Gyroscope LSM6DSOX
+* Authors:       William Lim
+*
+* Email:         wll180000@utdallas.edu
+*
+* (c) 2023 Dallas Formula Racing - Embedded Firmware Team
+* Formula SAE International Collegiate Chapter
+* GPL-3.0 License
+*/
+
+#ifndef GYROSCOPE_LSM6DSOX_H
+#define GYROSCOPE_LSM6DSOX_H
+
+// ST HAL Dependencies
+#include "i2c.h"
+
+// DFR Custom Dependencies
+#include "igyroscope.hpp"
+
+#include <stdint.h>
```

Comment:
I totally forgot the add the // standard libraries comment on this. Will do it next commit.